lib/intercom/service/conversation.rb

```diff
@@ -36,6 +36,11 @@ def close(reply_data)
         reply reply_data.merge(message_type: 'close', type: 'admin')
       end
 
+      def snooze(reply_data)
+        snoozed_until = reply_data.fetch(:snoozed_until) { fail 'snoozed_until field is required' }
+        reply reply_data.merge(message_type: 'snoozed', type: 'admin')
```

Comment:
HI @theandrewykim , sorry for the delay here, forgot to get back to this. I was going to merge this but I just had one further question ... should the snoozed_until value be passed in with the replay, similar to assignee_id in the assign? e.g.
`reply reply_data.merge(message_type: 'snoozed', type: 'admin', snoozed_until: snoozed_until)`
I could be missing something here, i was just comparing it to the assign. Let me know and I can merge it then

Reply:
I just checked this again and it looks like the assign is wrong, well not wrong but unnecessary, i.e. adding in the field in the merge after checking it exists in the fetch, so I think this is good